clients/src/main/java/org/apache/kafka/common/metrics/stats/TotalCount.java

```diff
@@ -14,25 +14,18 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.kafka.streams.processor.internals.metrics;
+package org.apache.kafka.common.metrics.stats;
 
-import org.apache.kafka.common.metrics.MeasurableStat;
 import org.apache.kafka.common.metrics.MetricConfig;
 
 /**
- * A non-SampledStat version of Count for measuring -total metrics in streams
+ * A non-sampled version of {@link Count} maintained over all time.
+ *
+ * This is a special kind of {@link Total} that always records {@code 1} instead of the provided value.
  */
-public class CumulativeCount implements MeasurableStat {
-
-    private double count = 0.0;
-
+public class TotalCount extends Total {
```

Comment:
Since we're relocating packages, it's a good opportunity to try an pick a more semantic and consistent name... Not sure if I succeeded.